frontend/src/components/dashboard/roles/ICTAdminSchoolManagement.tsx

```
@@ -113,7 +114,9 @@ const ICTAdminSchoolManagement: React.FC = () => {
       }
 
       const schoolParam = selectedSchoolId ? `?school_id=${selectedSchoolId}` : '';
-      const response = await fetch(`/api/v1/bulk-students/template${schoolParam}`, {
+      const baseUrl = ENV.API_BASE_URL || '/api/v1';
+      const url = `${baseUrl}/bulk-students/template${schoolParam}`;
+      const response = await fetch(url, {
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)